juniper_official/interface/check-ifl-state.rule

```diff
@@ -118,6 +216,16 @@ healthbot {
                 description "Sub interfaces index to be monitored, regular expression, eg '0-10'";
                 type string;
             }
+            variable traffic-high-threshold {
```

Comment:
Why are we using thresholds where there are no triggers? What is the porpose here? Only collecting the data without any evaluations????